### DISCUSSION TITLE
checkbox don't work

### DISCUSSION THREAD

**Discussion by maxiaojunwolf** [aminalaee/sqladmin#86] [General]
here is my ORM Field:
 
        picture = Column(Boolean, default=False)

![20220315-224244](https://user-images.githubusercontent.com/34365493/158403156-d0b319a7-1c87-4dd3-9733-f3dbd5a89b2f.png)

**Comment by aminalaee** [comment]
Hey, is this the create page or the edit page?

**Reply by maxiaojunwolf** [reply]
both

**Reply by aminalaee** [reply]
Thanks for reporting this. Issue and PRs created for this: https://github.com/aminalaee/sqladmin/issues/87
